### ISSUE TITLE
Slashes in entity field names break the generation

### ISSUE THREAD

**Issue by ramondebeijer** [mrbandler/cds2types#32]
Hi @mrbandler,

We have some entities generated from SAP tables that contain slashes in the name (I know why on earth....), 
so for example:

`
  entity Test {
    ![FIELD1] : String(10);
    ![/PART1/PART2] : String(23);
  };
`

This is currently breaking the cds2types, is there a way to fix this

kind regards 
Ramon

**Comment by HeneryHawk** [comment]
Hi @ramondebeijer 

i just released beta version 2.7.4-beta.0 which should solve the problem. Can you please test this and give feedback if this solves the problem? If so, I'll release version 2.7.4.

CDS definition
```
entity EntityWithSlashes {
    field1: String(10);
    ![/part1/part2]: String(23);
}
```
results in
```typescript
export interface IEntityWithSlashes {
    field1: string;
    "/part1/part2": string;
}
```

Resolved in commit: c9c61c05a77051e175b15071375e27e0d336881a

Regards
Simon

**Comment by ramondebeijer** [comment]
wow. this is mega fast thanks @HeneryHawk , I can confirm that it works!!!

**Comment by HeneryHawk** [comment]
Hi @ramondebeijer 

i just released version 2.7.4.

Regards
Simon

**Comment by ramondebeijer** [comment]
thanks!!
